unit_testing/cli/test_cli_sct_label_vertebrae.py

```diff
@@ -1,7 +1,8 @@
 from pytest_console_scripts import script_runner
 import pytest
 import logging
-
+import subprocess
```

Comment:
I don't think `subprocess` is needed anymore. :slightly_smiling_face:

Reply:
Yes right!

Reply:
Done :)